### PR TITLE
Update endpoints

### PR DESCRIPTION
**Pull request by alexisluque**

### CODE REVIEW

File: 01-Authorization/README.md [hunk 1]
```diff
@@ -22,44 +22,62 @@ mvn -v
 
 ## Configure the endpoints
 
-This seed uses two endpoints: `ping` and `secured/ping`. The former will not require authentication, while the later will do.
+This seed uses three endpoints: `api/public`, `api/private` and `api/private`. The former will not require authentication, while the later will do.
```
Comment (aaguiarz) [inline 01-Authorization/README.md]: It says api/private twice

**Review by aaguiarz — APPROVED** [review]

**Comment by lbalmaceda** [comment]
Shouldn't we add 3 tests to assert that the API endpoints actually respond?

**Review by paulioceano — COMMENTED** [review]
See issue #29

**Comment by aaguiarz** [comment]
@alexisluque can you review #29?

**Pull request merged**